README.md

```diff
@@ -50,6 +50,7 @@ List of helpful tmux links for various tutorials, plugins, and configuration set
 - [tmux-cssh](https://github.com/zinic/tmux-cssh) Tmux with a "ClusterSSH"-like behavior
 - [tmux-suspend](https://github.com/MunifTanjim/tmux-suspend) Suspend local session for painlessly working with nested remote session.
 - [tmux-up](https://github.com/jamesottaway/tmux-up) Bootstrap new `tmux` sessions without complex tools, DSLs, or dependencies
+- [tmuxake] (https://github.com/nkh/tmuxake) A side-pane manager for tmux
```

Comment:
There's a space after ] which makes the link not render properly. Screenshot here.

![image](https://user-images.githubusercontent.com/371796/128811238-23117aec-df96-496c-96c8-1986b19a1f03.png)

Reply:
updated